services/test-validators.js

```diff
@@ -26,6 +26,12 @@ const isVPlusDottedVersionNClausesWithOptionalSuffix = withRegex(
   /^v\d+(\.\d+)*([-+~].*)?$/
 )
 
+// same as above, but also accepts an optional 'epoch' prefix that can be
+// found e.g. in distro package versions
```

Comment:
Could you include in this comment an example of the kind of string you want to match? I think that will help people who are unfamiliar with these distro package versions.

Reply:
Done